openbook_posts/tests/views/test_post_comment_replies.py

```diff
@@ -1644,6 +1683,75 @@ def test_replying_on_foreign_post_comment_doesnt_send_push_notification_when_pos
 
         send_post_comment_reply_push_notification_call.assert_not_called()
 
+    def test_replying_on_post_comment_doesnt_create_push_notification_when_user_blocked(self):
+        """
+         should NOT create notification when replying on a post comment if user is blocked
```

Comment:
I don't get it. When the post creator blocks a user, the blocked user shouldn't even be able to comment on his posts. 

Shouldn't this test be related to the 

```
        post_creator.reply_to_comment_with_id_for_post_with_uuid(post_comment_id=post_comment.pk,
                                                                 post_uuid=post.uuid,
                                                                 text=make_fake_post_comment_text())
```

Instead of commenting with the blocked user on a post he should not be able to see anymore?

```
        url = self._get_url(post, post_comment)
        self.client.put(url, data, **headers)
```

The `post_creator.reply_to_comment_with_id_for_post_with_uuid` should be what happens on the `self.client.put`

I think because you copied and pasted this from the other tests, you missed this.

Reply:
Could we also change the name of the test a bit to make it explicit?

`should not create notification for blocked user when post creator replies to foreign post comment`

Reply:
Oh wait, yea i copied it but changed some things but forgot to change the names of the user... the post_creator is not the post_creator as you can see, its actually the foreign_user who creates the post.   Ill rename the users